packages/ui/package.json

```diff
@@ -35,7 +35,7 @@
     "jest": "^27.5.1",
     "prettier": "^2.6.0",
     "prop-types": "^15.8.1",
-    "react": "^17.0.2",
+    "react": "^16.0.0 || ^17.0.2",
```

Comment:
```suggestion
    "react": "^17.0.2",
```

Reply:
@kilemensi This was to fix the peer dependency issues with storybook-addon-materialui5